crates/flowctl/src/config.rs

```diff
@@ -1,27 +1,67 @@
+use anyhow::Context;
 use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+lazy_static::lazy_static! {
+    static ref DEFAULT_DASHBOARD_URL: url::Url = url::Url::parse("https://dashboard.estuary.dev/").unwrap();
+}
 
 #[derive(Debug, Serialize, Deserialize, Default)]
 pub struct Config {
+    /// URL of the Flow UI, which will be used as a base when flowctl generates links to it.
+    pub dashboard_url: Option<url::Url>,
     /// ID of the current draft, or None if no draft is configured.
     pub draft: Option<String>,
     // Current access token, or None if no token is set.
     pub api: Option<API>,
 }
 
 impl Config {
-    pub fn client(&self) -> anyhow::Result<postgrest::Postgrest> {
-        match &self.api {
-            Some(api) => {
-                let client = postgrest::Postgrest::new(api.endpoint.as_str());
-                let client = client.insert_header("apikey", &api.public_token);
-                let client =
-                    client.insert_header("Authorization", format!("Bearer {}", &api.access_token));
-                Ok(client)
+    /// Loads the config corresponding to the given named `profile`.
+    /// This loads from:
+    /// - $HOME/.config/flowctl/${profile}.json on linux
+    /// - $HOME/Library/Application Support/flowctl/${profile}.json on macos
```

Comment:
Is there a reason to use a different path on macos? I would find it much easier if I went looking for my flowctl config and found a `~/.flowctl` folder, rather than digging around in `~/Library/Application Support`, which I tend to think of as opaque and not very user-introspectable

Reply:
FWIW, my own intuition matches yours here. I'm pretty accustomed to using something like `~/.flowctl/` or `~/.config/flowctl/` on OSX, and I'm not aware of any problems with that. But the choice of which directory to use in this case is made by [`dirs::config_dir`](https://docs.rs/dirs/latest/dirs/fn.config_dir.html), and all I wanted to do in _this_ PR is just add some documentation on where that's looking. I could see changing it to `~/.config/flowctl/` in a future PR, but I'd want to check with a couple more mac users first.